pilot/pkg/kube/inject/testdata/traffic-params.yaml.injected

```diff
@@ -94,7 +94,7 @@ spec:
         - -x
         - 10.96.0.2/24,10.96.0.3/24
         - -b
-        - 1,2,3
+        - 80,
```

Comment:
guessing this is breaking ...

Reply:
This is in fact correct, not sure why the 1,2,3 was there before, there is no annotation.

Reply:
yes looks like you're right, looks like the master version was wrong